packages/core-mobile/app/services/swap/SwapService.ts

```diff
@@ -108,30 +108,22 @@ class SwapService {
         if (!account) {
           throw new Error('Account address missing')
         }
-
-        const optimalRates = async (): Promise<OptimalRate | Error> => {
-          return await this.getParaSwapSDK(network.chainId).swap.getRate({
-            srcToken,
-            destToken,
-            amount: srcAmount,
-            userAddress: account.addressC,
-            side: swapSide,
-            srcDecimals,
-            destDecimals
-          })
-        }
-
-        return await retry({
-          operation: optimalRates,
-          isSuccess: result => (result as Error).message !== 'Server too busy'
```

Comment:
so does it mean that we don't have to retry with the new function?

Reply:
Not sure why we needed that in the first place.  we don't simply retry whenever we can't fetch because `Server was busy` in this case.  we should just let it error out, and user can retry manually.